packages/open-next/src/plugins/resolve.ts

```diff
@@ -26,6 +32,9 @@ export interface IPluginSettings {
       | LazyLoadedOverride<OriginResolver>
       | IncludedOriginResolver;
     warmer?: LazyLoadedOverride<Warmer> | IncludedWarmer;
+    proxyExternalRequest?:
```

Comment:
nit: 
- use `OverrideOptions["..."]` for `warmer` and `proxyExternalRequest`
- `DefaultOverrideOptions<any, any>` can also be changed for `OverrideOptions`, right?

Reply:
> DefaultOverrideOptions<any, any> can also be changed for OverrideOptions, right?

Yes

> OverrideOptions["..."] for warmer and proxyExternalRequest

We can't for the warmer, it is not defined in OverrideOptions

Reply:
Right I mixed with wrapper 🙃